superset-frontend/src/explore/controlPanels/Bar.js

```diff
@@ -29,39 +49,33 @@ export default {
       expanded: true,
       controlSetRows: [
         ['color_scheme', 'label_colors'],
-        ['show_brush', 'show_legend', 'show_bar_value'],
-        ['rich_tooltip', 'bar_stacked'],
-        ['line_interpolation', 'show_controls'],
-        ['bottom_margin'],
+        [showBrush, showLegend, showBarValue],
+        [richTooltip, barStacked],
+        [lineInterpolation, showControls],
+        [bottomMargin],
       ],
     },
     {
       label: t('X Axis'),
       expanded: true,
       controlSetRows: [
-        ['x_axis_label', 'bottom_margin'],
-        ['x_ticks_layout', 'x_axis_format'],
-        ['x_axis_showminmax', 'reduce_x_ticks'],
+        [xAxisLabel, bottomMargin],
+        [xTicksLayout, xAxisFormat],
```

Comment:
This one also did not apply override to `xAxisFormat`

Reply:
Same as above... I swapped the defaults/overrides, since there were more _with_ identical overrides than without.